arelle/plugin/validate/EDINET/rules/edinet.py

```diff
@@ -888,12 +888,52 @@ def rule_EC8073E(
             if any(illegalChars):
                 yield Validation.error(
                     codes='EDINET.EC8073E',
-                    msg=_("The concept: %(concept) has a %(role) label which contains characters that are not "
-                          "allowed. Label: %(label), disallowed characters: %(characters)"),
+                    msg=_("The concept: %(concept)s has a %(role)s label which contains characters that are not "
+                          "allowed. Label: %(label)s, disallowed characters: %(characters)s"),
                     concept=rel.toModelObject.qname.localName,
                     role=labelRel.toModelObject.role,
                     label=labelRel.toModelObject.textValue,
-                    characters=illegalChars
+                    characters=list(illegalChars)
+                )
+
+
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+    disclosureSystems=[DISCLOSURE_SYSTEM_EDINET],
+)
+def rule_EC8073W(
```

Comment:
Could this just be rolled into EC8073E? It seems like it's a warning if it's related to `executiveOfficersMemberQn` but an error otherwise. Check out `EC8060E` for reference.